clients/js/src/consts/networks.ts

```diff
@@ -166,7 +166,7 @@ const MAINNET = {
   },
   sei: {
     rpc: undefined,
```

Comment:
I think there's a public RPC now, yes?

Reply:
Yup, I took one from [here](https://docs.sei.io/develop/resources#public-rpcs) for `pacific-1`.